fontData.js

```diff
@@ -192,6 +199,21 @@ const buildChars = font => {
     }
   }
 
+  // List all chars not grouped under scripts in a misc category
+  // Also, ignore 0xFFFF which is erroneously reported as a char
+  // by Fontkit
```

Comment:
```suggestion
  // by Font.js
```

Reply:
That's true for the _other_ project ;-) We're still on Fontkit here.

(But I'd more and more love to use the Wakamai Fondue engine here!!)

Reply:
Oh haha yes you're right, I'm mixing things up here 😅